src/main/java/client/interface_adapters/controllers/change_profile/ChPrController.java

```diff
@@ -0,0 +1,67 @@
+package client.interface_adapters.controllers.change_profile;
+
+import client.frameworks_and_drivers.communication_manager.IfComManager;
+import client.interface_adapters.Constants;
+import client.interface_adapters.model.Model;
+
+public class ChPrController implements ChPrControllerInputBoundary {
+
+    int serverport = 4396;
+    private final IfComManager comManager;
+    private final Model model;
+
+    char CONTENT_SPR = Constants.SPR;
+    String HASH ="#";
+    private final String serverIp;
+
+    public ChPrController(IfComManager comManager, Model model, String serverIp)
+    {
+        this.comManager = comManager;
+        this.model = model;
+        this.serverIp = serverIp;
+    }
+
+    @Override
+    public void setPic(String content) {
+        int selfUid = model.getSelfUid();
+        String toSend = String.valueOf(server.frameworks_and_drivers.Constants.SET_PIC + HASH + selfUid
+                +
+                CONTENT_SPR+"DONTCHANGE"
+                + CONTENT_SPR+"DONTCHANGE"
+                +CONTENT_SPR + content);
+
+        comManager.send(serverIp, serverport, toSend);
+    }
+
+    @Override
+    public void delPic() {
+        int selfUid = model.getSelfUid();
+        String toSend = String.valueOf(server.frameworks_and_drivers.Constants.DEL_PIC + HASH + selfUid
```

Comment:
for now yes, because here controller sends the message to the server and the server will sort by its constants

maybe we can use only one class of that constants for both side or have two identical but be sure numbers there are the same